hack/verify-canonical-import-path.sh

```diff
@@ -0,0 +1,50 @@
+#!/usr/bin/env bash
+
+# Copyright 2018 The Kubernetes Authors.
```

Comment:
I'm truly baffled why this boilerplate is wrong. I used the same one as that of another script in hack/.

```
➜  kubernetes git:(verify-canonical-import-path) hack/verify-boilerplate.sh                                                                                         
Boilerplate header is wrong for: /home/nikhita/go/src/k8s.io/kubernetes/hack/verify-canonical-import-path.sh
```

Am I missing something here?